tests/unit/visual/tst_tabs.13.qml

```diff
@@ -116,7 +116,7 @@ MainView {
     Component {
         id: pageComponentNoFlick
         Page {
-            title: "Loaded page without flickable"
+            header: PageHeader { title: "Loaded page without flickable" }
```

Comment:
Is the old code supposed to work?

Reply:
I don't know. It results in several deprecation warnings, as well as resulting in the height being different causing a test failure, without the change.